src/components/ActionPanel/index.tsx

```diff
@@ -211,13 +211,14 @@ function ActionPanel(props: Props) {
       //     />
       //   );
       case WITHDRAW:
+        return (
```

Comment:
`BabylonDelegationHelper.verifyDestination` would determine if this UI should be displayed on not based on contract address.